src/OpenFeature.Contrib.Providers.Flagd/Resolver/InProcess/JsonEvaluator.cs

```diff
@@ -136,16 +138,31 @@ private ResolutionDetails<T> ResolveValue<T>(string flagKey, T defaultValue, Eva
                 var variant = flagConfiguration.DefaultVariant;
                 if (flagConfiguration.Targeting != null && !String.IsNullOrEmpty(flagConfiguration.Targeting.ToString()) && flagConfiguration.Targeting.ToString() != "{}")
                 {
+                    var flagdProperties = new Dictionary<string, Value>();
+                    flagdProperties.Add(FlagdProperties.FlagKeyKey, new Value(flagKey));
+                    flagdProperties.Add(FlagdProperties.TimestampKey, new Value(DateTime.Now));
```

Comment:
I think `DateTime.New` returns a DateTime object. In order for this to work with flag rules, we need to inject a unix timestamp (see spec).

_I think_ you want something more like: `DateTimeOffset.UtcNow.ToUnixTimeSeconds()`, but please verify. Also, add a unit test if it's not too hard (if it is, or you think the ROI is low, don't worry since we have an e2e test specifically for this).

Reply:
alright, I have adapted it accordingly and added a simple test to check if a timestamp is present and can be used within a targeting rule :)